Penteract/Assets/Scripts/AIMovement.cpp

```diff
@@ -105,6 +106,22 @@ void AIMovement::Update() {
             if (hudControllerScript) {
                 hudControllerScript->UpdateScore(10);
             }
+            /*GameObject* spawn = GetOwner().GetParent();
+            if (spawn != nullptr) {
+                EnemySpawnPoint* enemyspawnpoint = GET_SCRIPT(spawn, EnemySpawnPoint);
+                if (enemyspawnpoint != nullptr) {
+                    //Debug::Log("ENEMY DEATH TO SPAWN");
+                    enemyspawnpoint->KillEnemy();
+                }
+            }*/
+            GameObject* winlose = GameplaySystems::GetGameObject("WinCon");
```

Comment:
Expose the GameObjectUID to get the game object via UID with a cost of O(1), instead of search by name that has a cost of O(n), with n = number of game objects in the scene. To expose this UID, you have the playerUID as an example.